parla/dataflow.py

```diff
@@ -4,10 +4,10 @@
 and also eager-fashion automatic data movement.
 """
 
-from typing import Collection, Any
+from typing import List, Any
 from itertools import chain
 
-from parla.task_runtime import get_current_devices
+from parla import task_runtime
```

Comment:
Why the whole module? Is more than `get_current_devices` used?

Reply:
For avoiding circular import. I added more imports for type hints.